src/components/Psychologist/Dates/Dates.jsx

```diff
@@ -18,7 +26,7 @@ function Dates() {
 	const AuthStr = `Bearer ${data.token}`;
```

Comment:
Why you send the AuthStr from the component instead of service?

Reply:
This part is of the changes that I came down from my partner Eduardo